docs/source/how-tos/create_an_openedx_app.rst

```diff
@@ -1,14 +1,28 @@
 Create an Open edX Application for the Plugin Settings
 =======================================================
 
-You will learn how to create an Open edX Application for filling out the form in the Open edX Sync Plugin Settings in our Wordpress Settings.
+You will learn how to create an Open edX Application for filling out the form in the Open edX Sync Plugin Settings in our WordPress Settings.
+
+Index
+-------
+- `Requisites`_
+- `Steps`_
+- `Next Steps`_
+
+Requisites
+-----------
+
+Have access to a Django admin dashboard for your Open edX platform.
+
+Steps
```

Comment:
I already updated this article with your suggestions but didn't understand this: "How to Set Up the WordPress Plugin for Open edX Commerce."

Are you suggesting it is better to change the article's title or "Steps" subtitle to that? If so, I'm not sure because most of the steps are around creating an app (that is a requisite to configure) and not configuring itself. But if you still think it is better, I can change it. Please let me know what you suggest.

Reply:
I changed the "Steps" to "Create an Open edX Application to Configure the Open edX Commerce plugin."